src/Commands/SSHKey/AddCommand.php

```diff
@@ -24,7 +24,11 @@ class AddCommand extends TerminusCommand
      */
     public function add($file)
     {
-        $this->session()->getUser()->getSSHKeys()->addKey($file);
+        $response = $this->session()->getUser()->getSSHKeys()->addKey($file);
+        if ($response['status_code'] !== 200) {
+            $this->log()->error($response['data']);
```

Comment:
Can this messaging be improved to explicitly state that `ed25519` keys are "not yet supported"?

I see BUGS-5401 says expected behavior is "An error is thrown stating ed25519 keys are not supported", and this gets the error part done, but my sense is that the messaging could be a bit clearer..

Reply:
+1 to Brian's note :) "ssh-ed25519 is not yet supported" would be helpful.